third-party/json/json11.cpp

```diff
@@ -69,6 +70,22 @@ static void dump(int value, string &out) {
     snprintf(buf, sizeof buf, "%d", value);
     out += buf;
 }
+static void dump(unsigned int value, string &out) {
+    char buf[32];
```

Comment:
32 characters is probably a bit of an overkill here :) Besides: I can see ore and more places in our code that could benefit from using the [fmtlib](https://github.com/fmtlib/fmt)

Reply:
so maybe it's worth using it systemwide :P

Reply:
Yes, that's my intent - we could introduce it at last...

Reply:
Or just enable C++20 support for MuditaOS:)

Reply:
that's a great idea, but it's not so easy to do so - we can talk on internal channels though :)